### ISSUE TITLE
[a11y] AMP install banner appears near top of page in linear swipe order

### ISSUE THREAD

**Issue by dmochak** [ampproject/amphtml#4935]
When navigating linearly with a screen reader, the install banner is presented above the page content rather than below it.

## How do we reproduce the issue?

1. Activate TalkBack (settings > accessibility > TalkBack)
2. Open https://amp-mk-3.herokuapp.com/examples/viewer.html and open the install banner.
3. swipe to the right with one finger repeatedly.

Actual result: the banner comes into focus after the header links but before the content of the page.

expected result: the banner's placement in the navigation order should be in the same place it is on the screen. If the banner is on the bottom of the page, it should appear near the end.

## What browsers are affected?
Nexus 5X, Chrome, Android N.

## Which AMP version is affected?

Powered by AMP ⚡ HTML – Version 1473296795712

**Comment by ericlindley-g** [comment]
cc'ing @mkhatib 

This is an interesting case—the banner is at the bottom of the screen in the first viewport, but essentially sits above the content below the first viewport. How are fixed-position elements like this usually handled (for instance, cookie permission notices)? If the banner is after the page content, how many extra swipes does it typically take to get to the banner?

**Comment by mkhatib** [comment]
I am not sure I understand this correctly. But this might just be because I put the code for the amp-banner on the top of page in the example, this can be easily moved to the bottom of the page if that's the idea.

**Comment by ericlindley-g** [comment]
That makes sense—may be worth testing to see if that works, and then we can include a recommendation in a11y guidelines in the documentation.

**Comment by ericlindley-g** [comment]
Though actually, it may make more sense to keep the banner before the content, even if it visually is below it in the first viewport (often these app install banners sit above the content anyway). You wouldn't want the user to have to swipe through the entire article to get to the banner, so unless there's some other solution we can keep it how it is. What do you think?

**Comment by mkhatib** [comment]
I don't really have a preference but keeping it to the top sounds good to me. This is overall would be a decision for the publishers to decide where to put the code of the banner. Should we close this bug?

**Comment by ericlindley-g** [comment]
Agreed—thanks!
